gen/calc.py

```diff
@@ -296,7 +366,18 @@ def calculate_exhibitor_static_ensemble(master_list):
         'ui_banner_header_content': 'null',
         'ui_banner_footer_content': 'null',
         'ui_banner_image_path': 'null',
-        'ui_banner_dismissible': 'null'
+        'ui_banner_dismissible': 'null',
+        'dcos_overlay_network': '{                    \
```

Comment:
This doesn't seem particularly safe to default. Probably we want to just lay out in docs waht all needs to be set and how to set it. That this is not configurable at cluster runtime is likely to also be an issue.

As a code style thing, if we want the multi-line format here, should use a python multi-line comment `"""` and declare the value earlier, then just reference that declared value here.